client/Android/Studio/freeRDPCore/src/main/res/values-de/strings.xml

```diff
@@ -119,9 +119,9 @@
         <item>Auf Client abspielen</item>
     </string-array>
     <string-array name="redirect_sound_values_array">
-        <item>0</item>
-        <item>1</item>
         <item>2</item>
+        <item>1</item>
+        <item>0</item>
```

Comment:
Not sure why we think the indentation is wrong for here. 4 space before ''<string-array name="redirect_sound_values_array">' and 8 space before items seems to be reasonable:

  0001700: 2020 2020 3c73 7472 696e 672d 6172 7261      <string-arra
  0001710: 7920 6e61 6d65 3d22 7265 6469 7265 6374  y name="redirect
  0001720: 5f73 6f75 6e64 5f76 616c 7565 735f 6172  _sound_values_ar
  0001730: 7261 7922 3e0a 2020 2020 2020 2020 3c69  ray">.        <i
  0001740: 7465 6d3e 323c 2f69 7465 6d3e 0a20 2020  tem>2</item>.